po/pt-rBR.po

```diff
@@ -13,25 +13,25 @@ msgstr ""
 
 #: src/pages/About.vue:4 src/pages/Register.vue:5
 msgid "A cross-plattform signal client"
-msgstr ""
+msgstr "Um cliente multiplataforma do Signal"
 
 #: src/pages/Settings.vue:27
 msgid "About Axolotl"
-msgstr ""
+msgstr "Sobre Axolotl"
 
 #: src/components/AddContactModal.vue:5 src/components/AddContactModal.vue:22
 msgid "Add"
-msgstr ""
+msgstr "Adicionar"
 
 #: src/components/AddGroupMembersModal.vue:51
 #, fuzzy
```

Comment:
Thanks for your translations, it's really appreciated. In order to pick up the strings, the `#, fuzzy` needs to be removed, too, can you to this before we merge it?